NAMESPACE

```diff
@@ -37,6 +40,8 @@ export(getPopulation)
 export(getPopulations)
 export(getScaleSets)
 export(getStatistics)
+export(scaleFlowCoreGate)
+export(scaleSetToTransformList)
```

Comment:
Same for these, don't need to be exported?

Reply:
I intended these as features, more than as internal utility functions. `scaleFlowCoreGate` may not be necessary, since a CE gate could be converted to a flowCore gate, the `scaleSet` could be converted to a `transformList`, and then the `transformList` applied to a the flowCore gate.

Reply:
(See above comment - I'm in favor of one big method.)